### PR TITLE
docs: update troubleshooting

### PR DESCRIPTION
**Pull request by KazuCocoa**
Move a few missing things from https://appium.io/docs/en/drivers/ios-xcuitest/#accessibility-preferences-fine-tuning which might be good for troubleshooting

### CODE REVIEW

File: docs/troubleshooting.md [hunk 1]
```diff
@@ -56,4 +56,31 @@ In short, this driver tries to leave things as it found them.
 You can use the `noReset` capability to adjust this behavior.
 Setting `noReset` to `true` will leave the simulator running at the end of a test session.
 
+## Files generated by test runs
```
Comment (mykola-mokhnach) [inline docs/troubleshooting.md:59]: Probably we can move some of below topic under "device preparation"

Comment (KazuCocoa) [reply docs/troubleshooting.md]: makes sense. I've created the one, and added a few such as Web Inspector as well. Maybe... we still missed something (I haven't confirmed well yet)

File: docs/device-preparation.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
+---
+title: Device Preparation
+---
+
+## Automatic device preference update
+
+Appium XCUITest prepares the preference of the device under test automatically.
+Some are also configured over Appium capabilities.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:8]: Some of them are also configurable via ....

File: docs/device-preparation.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
+---
+title: Device Preparation
+---
+
+## Automatic device preference update
+
+Appium XCUITest prepares the preference of the device under test automatically.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:7]: prepares preferences

File: docs/device-preparation.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+Some are also configured over Appium capabilities.
+
+### keyboards configuration
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:10]: Keyboards

File: docs/device-preparation.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Appium configures keyboard preferences by default to make test running more stable on a real device.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:12]: to make test runs more stable

File: docs/device-preparation.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+You can change some of them via settings API.
+
+- Turn `Auto-Correction` in Keyboards off
+- Turn `Predictive` in Keyboards off
+- Mark keyboard tutorial as complete
+- (Only for Simulator) Toggle software keyboard on
+
+## Manual setup for fine tuning
+
+Automatic configuration availability is limited on iOS, especially for a real device. You may need to manually configure the device configuration.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:22]: to manually change the ...

File: docs/device-preparation.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+
+### Enable Web testing availability
+
+Please turn on _Web Inspector_ on iOS device via _Settings_ -> _Safari_ -> _Advanced_
+
+### Avoid possible wrong coordinate
+
+Please make sure the zoom preference in accessibility is turned off via _Settings_ -> _Accessibility_ -> _Zoom_. It could cause wrong element coordinate.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:30]: It could cause element coordinates miscalculation.

File: docs/device-preparation.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+### Expose more elements if needed
+
+In some cases, enabling of the below preferences helps to make some view elements accessible. Appium does not modify these settings automatically, since they could affect the way your application under test performs. Please change them manually if needed.
+
+- Turn `Spoken Content` in Settings > Accessibility on
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:36]: should we follow the common style while making Settings paths italic?

**Comment by KazuCocoa** [comment]
updated ✅

File: docs/device-preparation.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+
+## Automatic device preference update
+
+Appium XCUITest prepares preferences of the device under test automatically.
+Some of them are also configurable via Appium capabilities and the Settings API.
+
+### Keyboards configuration
+
+Appium configures keyboard preferences by default to make test runs more stable.
+You can change some of them via settings API.
```
Comment (mykola-mokhnach) [inline docs/device-preparation.md:13]: lets use a single approach to write `Settings API` everywhere

File: docs/setup.md [hunk 10]
```diff
@@ -70,4 +70,3 @@ Read the [tvOS support](ios-tvos.md) article to get more details on how to autom
 ### Configuration
 
 See the [real device configuration documentation](./real-device-config.md).
```
Comment (mykola-mokhnach) [inline docs/setup.md:72]: revert?

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**